### PR TITLE
Vendor specific table created with a new subgroup, Vendor_specific_Si…

### PR DESCRIPTION
**Pull request by SverreBerg**
…mradWBT, to provide transducer and transceiver information for processing and reference.

### CODE REVIEW

File: docs/tableVendor_specific_SimradWBT_sub_group.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ 3+|{attr}:units�=�"ohm" 
+ 
+ |{var}float transceiver_impedance_r(start_time, beam, frequency) |M |Transceiver impedance real part.
+ 3+|{attr}:long_name�=�"Transceiver impedance real part" 
+ 3+|{attr}:units�=�"ohm" 
+
+ |{var}int transceiver_sample_frequency(start_time, beam) |M |Transceiver sample frequency.
+ 3+|{attr}:long_name�=�"Transceiver sample frequency" 
+ 3+|{attr}int :valid_min�=�1
+
```
Comment (gavinmacaulay) [inline docs/tableVendor_specific_SimradWBT_sub_group.adoc:83]:
```suggestion
3+|{attr}:units = "Hz"

```

File: docs/tableVendor_specific_SimradWBT_sub_group.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+ |{var}float transceiver_impedance_i(start_time, beam, frequency) |M |Transceiver impedance imaginary part.
+ 3+|{attr}:long_name�=�"Transceiver impedance imaginary part" 
+ 3+|{attr}:units�=�"ohm" 
+ 
+ |{var}float transceiver_impedance_r(start_time, beam, frequency) |M |Transceiver impedance real part.
+ 3+|{attr}:long_name�=�"Transceiver impedance real part" 
+ 3+|{attr}:units�=�"ohm" 
+
+ |{var}int transceiver_sample_frequency(start_time, beam) |M |Transceiver sample frequency.
```
Comment (gavinmacaulay) [inline docs/tableVendor_specific_SimradWBT_sub_group.adoc:80]:
```suggestion
 |{var}int transceiver_sample_frequency(start_time, beam) |M |Transceiver sampling frequency.
```

**Review by gavinmacaulay — COMMENTED** [review]
Should the obligations be O instead of M?

**Review by SverreBerg — COMMENTED** [review]
Thanks a lot Gavin for valuable input.
We appreciate your assistance in making this a consistens contribution to the convention.
Will you help me renaming the Simrad specific file and update the reference to the file?

**Review by SverreBerg — COMMENTED** [review]
I edited the tableVendor_specific_SimradWBT_sub_group.adoc using the web.
Did that come through?

**Comment by gavinmacaulay** [comment]
> I edited the tableVendor_specific_SimradWBT_sub_group.adoc using the web. Did that come through?

I'm not sure - two of my suggested changes were marked as resolved, but the text wasn't changed (or did you just not like the suggestions ;). Apart from those, it all looks ok.

**Comment by SverreBerg** [comment]
I appreciate all suggestions :-) thanks.
Not sure which ones I didn't manage to include.
Filter gain is not in dB, could be one suggestion left out. (We normally try to make this 1 in our filters)
Filter generator version has been changed to a string, as suggested.

**Comment by gavinmacaulay** [comment]
The two things were the introductory text and the name of the subgroup (changed to SimradWBT). I found that I could commit those changes, so have done so. Have look and check that it looks ok.

**Comment by SverreBerg** [comment]
Thanks Gavin, your changes look good.
Anything more I need to do to complete this proposal?

**Pull request merged**